crates/blockifier/src/execution/entry_point.rs

```diff
@@ -185,23 +191,29 @@ macro_rules! retdata {
 }
 
 #[derive(Debug, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "parity-scale-codec", derive(Encode, Decode))]
 pub struct OrderedEvent {
+    #[cfg_attr(feature = "parity-scale-codec", codec(skip))]
```

Comment:
Don't skip it.
Go for a manual impl where you cast it as u64.

Also, if you look at the code created by `derive(Encode)` you will see it also `impl EncodeLike for MyType {}`, so if you can do the same on all of your manual encode it would be great (but not necessary for what I have seen so far)